website/blog/2022-06-30-aluminium.md

```diff
@@ -0,0 +1,341 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.7 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.7, which brings a large
+number of Scala 3 improvements as well as a few interesting code actions.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">149</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">67</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">7</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/51?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the [Scala Center](https://scala.epfl.ch/)
+and [VirtusLab](https://virtuslab.com) with the help from
+[Lunatech](https://lunatech.com) along with contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Release notes now shown in VS Code
+- Improvements to Bloop Java home automation.
+- [Scala 3] Implement-all abstract members code action.
+- [Scala 3] Add snippet completions.
+- [Scala 3] Add file name completions.
+- [Scala 2] Add ConvertToNamedArguments code action.
+- New flatMap to for comprehension code action.
+- added support for Scala 2.12.16, 3.1.3 and 3.2.0-RC1
+
+## Release notes now shown in VS Code
+
+If you are using Visual Studio Code you should see this message right after
+update of the Metals extension has finished. 🚀
+
+Starting with the current release we will be showing release notes inside VS
+Code when the version update completes. This way users will know whenever the
+extension was updated and it will not just silently update in the background.
```

Comment:
```suggestion
extension was updated and it will not just silently update in the background.

Release notes will be shown only once after update, but can be viewed again by executing `Metals: Show release notes`
```

Reply:
Changed, thanks!